src/media/css/app-list.styl

```diff
@@ -129,6 +130,20 @@ $desktop-tile-padding = 20px 50px;
             padding-top: 30px;
         }
     }
+    // Colored grids.
+    .app-list-colored {
+        .app-list-colored:after {
```

Comment:
Nope, copy-pasta when it lived in feed/colors.styl. The one there still works this one is not needed.